app/code/Magento/Ui/Model/Export/ConvertToCsv.php

```diff
@@ -77,7 +77,8 @@ public function getCsvFile()
         $this->directory->create('export');
         $stream = $this->directory->openFile($file, 'w+');
         $stream->lock();
-        $stream->writeCsv($this->metadataProvider->getHeaders($component));
+        $headers = array_map(array($this, 'removeQuote'), $this->metadataProvider->getHeaders($component));
```

Comment:
Please, use the anonymous function instead of declaring the new public method.

Reply:
Hello @sidolov,

I have remove array_clean function and use anonymous function instead of separate function.

Please check/review the changes and let me know if further inputs is required from my-side.

 Thanks 
Nirav Patel